### PR TITLE
French Uru credits in alignment with English contents

### PR DESCRIPTION
**Pull request by ZarothYe**
This PR brings the French credits journal contents (previously copied from Uru:CC credits, I believe) into alignment with the MOUL Uru credits that we have in English. I welcome any feedback on any French that needs to be adjusted. I made my best guesses based on existing CC credit contents for wording on titles/roles along with a sprinkling of Google Translate. There are still a few snippets in the Licences Open Source section that are in English if anyone wants to take a crack at writing those in French. I felt highly underqualified to try those myself. ;)

This PR also fix very minor spacing abnormalities in the English credits.

Videos of the results in-game.

English:

https://user-images.githubusercontent.com/94947971/195764911-b54d0f71-199d-4b72-b0c7-311805fd855c.mp4

French:

https://user-images.githubusercontent.com/94947971/195764960-bc07ad28-6485-4106-9159-4268d8c6bd55.mp4

### CODE REVIEW

File: compiled/dat/GlobalFrench.loc [hunk 1]
```diff
@@ -185,332 +185,200 @@
...
+<font size=24 spacing=-10><p align=left>Rand Miller<p align=right>Directeur
+
+<p align=center>Programmation<font spacing=-5>
+<font size=24 spacing=-10><p align=left>Mark H. DeForest<p align=right>CTO
+<p align=left>Bob Zasio<p align=right>Programmeur graphique principal
+<p align=left>Eric S. Anderson<p align=right>Programmeur serveur principal
+<p align=left>Jeff Lundin<p align=right>Programmation du client
+<p align=left>Jason M. Calvert<p align=right>Programmation du client
+<p align=left>Adam Van Ornum<p align=right>Programmation du client
+<p align=left>Randy Ridgway<p align=right>Programmeur physiques
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:198]: I think this should be Programmeur physique, no "s"

File: compiled/dat/GlobalFrench.loc [hunk 2]
```diff
@@ -185,332 +185,200 @@
...
-<p align=left>Bryce J. Biggerstaff<p align=right>Testeur
-<p align=left>Tim S. DeWolf<p align=right>Testeur
-<p align=left>Brandon A. Knowles<p align=right>Testeur
-<p align=left>Bryan Bussard<p align=right>Testeur
-<p align=left>Derek Odell<p align=right>Testeur
-<p align=left>Zachary M. Wellsandt<p align=right>Testeur
-<p align=left>Tyler J. Smith<p align=right>Testeur
-<pb><font size=26><p align=center>Conception et commandes<font spacing=-5>
+
+<font size=26><p align=center>Conception et commandes<font spacing=-5>
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:216]: I believe control in this case might be better translated to "contrôle".

File: compiled/dat/GlobalFrench.loc [hunk 3]
```diff
@@ -185,332 +185,200 @@
...
+<p align=left>Eliza Wyatt<p align=right>Testeuse
+
 <font size=26><p align=center>Support administratif<font spacing=-5>
 <font size=24 spacing=-10><p align=left>Tony Fryman<p align=right>Président
 <p align=left>Mark Dobratz<p align=right>Chef de projet
 <p align=left>Susan R. Bonds<p align=right>Co-Productrice
-<p align=left>Chris Brandkamp<p align=right>VP Business
-<p align=left>John M. Biggs<p align=right>Responsable des systèmes
 <p align=left>Mark Klammer<p align=right>Directeur financier
+<p align=left>Dave Smith<p align=right>Responsable du informatique
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:249]: Should be "Responsable de l'informatique"

File: compiled/dat/GlobalFrench.loc [hunk 4]
```diff
@@ -185,332 +185,200 @@
...
-<p align=left>Mark Engberg/Colab<p align=right>Conception de l'architecture
-<p align=left>Thom Schillinger<p align=right>Conception graphique
-<p align=left>Lee Sheldon<p align=right>Conception de l'histoire
-<p align=left>Gil Keppler<p align=right>Conception graphique
-<font spacing=-18>
-<font size=26><p align=center>Acteurs voix<font spacing=-5>
-<font size=24 spacing=-10><p align=left>David Ogden Stiers<p align=right>Zandi
-<p align=left>Rengin Altay<p align=right>Yeesha
+
+<font size=26 spacing=-18><p align=center>Acteurs voix<font spacing=-5>
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:253]: Voice actor is "Doubleur", or more elegantly "Artistes Vocal"

File: compiled/dat/GlobalFrench.loc [hunk 5]
```diff
@@ -185,332 +185,200 @@
...
+<pb><font spacing=-18>
+<font size=24 spacing=-10><p align=center>libepoxy
+Copyright (c) 2013-2014 Intel Corporation
+https://github.com/anholt/libepoxy
+
 <font spacing=-18>
-<font size=24 spacing=-10><p align=center>Free Type
-Le projet FreeType est sous copyright (c) 1996-2000 par David Turner, Robert Wilhelm et Werner Lemberg. Tous droits réservés.
+<font size=24 spacing=-10><p align=center>libjpeg-turbo
+This software is based in part on the work of the Independent JPEG Group.
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:318]: Ce logiciel est basé en partie sur le travail de l'Independent JPEG Group.

File: compiled/dat/GlobalFrench.loc [hunk 6]
```diff
@@ -185,332 +185,200 @@
...
+<font spacing=-18>
+<font size=24 spacing=-10><p align=center>openal-soft
+https://github.com/kcat/openal-soft
+
 <font spacing=-18>
-<font size=24 spacing=-10><p align=center>Boost
-Copyright (c) 2002 CrystalClear Software, inc. La permission d'utiliser, de copier, de distribuer et de vendre ce logiciel, quelles qu'en soient les fins, est accordée sans frais.
+<font size=24 spacing=-10><p align=center>OpenSSL
+Copyright (c) 1998-2019 The OpenSSL Project. Tous droits réservés.
+This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit (http://www.openssl.org/)
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:354]: Ce produit inclut un logiciel développé par le OpenSSL Project pour une utilisation dans OpenSSL Toolkit

File: compiled/dat/GlobalFrench.loc [hunk 7]
```diff
@@ -185,332 +185,200 @@
...
+This product includes cryptographic software written by Eric Young (eay@cryptsoft.com).
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:355]: Ce produit comprend un logiciel cryptographique écrit par Eric Young

File: compiled/dat/GlobalFrench.loc [hunk 8]
```diff
@@ -185,332 +185,200 @@
...
+This product includes software written by Tim  Hudson (tjh@cryptsoft.com).
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:356]: Ce produit comprend un logiciel écrit par Tim Hudson

**Review by skyisbluu — CHANGES_REQUESTED** [review]

**Comment by dgelessus** [comment]
> There are still a few snippets in the Licences Open Source section that are in English if anyone wants to take a crack at writing those in French.

Hmm, I wonder if the license/attribution bits should be translated at all? Because for example the SSLeay license says:

> 3. All advertising materials mentioning features or use of this software must display the following acknowledgement:
>    "This product includes cryptographic software written by Eric Young (eay@cryptsoft.com)"
>    The word 'cryptographic' can be left out if the rouines from the library being used are not cryptographic related :-).

I'm not a lawyer obviously, but since the license asks for this specific sentence, I feel like it would be safer to leave it untranslated..

**Comment by Hoikas** [comment]
> > There are still a few snippets in the Licences Open Source section that are in English if anyone wants to take a crack at writing those in French.
> 
> Hmm, I wonder if the license/attribution bits should be translated at all? Because for example the SSLeay license says:
> 
> > 3. All advertising materials mentioning features or use of this software must display the following acknowledgement:
> >    "This product includes cryptographic software written by Eric Young ([eay@cryptsoft.com](mailto:eay@cryptsoft.com))"
> >    The word 'cryptographic' can be left out if the rouines from the library being used are not cryptographic related :-).
> 
> I'm not a lawyer obviously, but since the license asks for this specific sentence, I feel like it would be safer to leave it untranslated.

I had the same reaction initially, but I am leaning more toward wanting the notice to be translated for each individual localization. The credits here are a user friendly listing and do not necessarily represent the end-all-be-all of open source licenses. Ideally, we keep them up to date, but it's definitely a manually curated process. We really should have an untranslated definitive list of library credits produced using eg vcpkg's `share/{$PORT}/copyright` files lying in the game directory, which would contain the complete and unadulterated list of credits. That's, of course, outside the scope of this PR.

I'd appreciate comments from other maintainers on this.

**Comment by ZarothYe** [comment]
New French preview with the updates and new translations from sky included :)

https://user-images.githubusercontent.com/94947971/196092275-62c8154b-a6e6-4d9d-8c34-41444dc16bc6.mp4

**Comment by dpogue** [comment]
Merci Zaroth et Skyisblu pour vos efforts ici! 🙇🏼‍♂️

File: compiled/dat/GlobalFrench.loc [hunk 9]
```diff
@@ -185,332 +185,200 @@
...
-<p align=left>Jason M. Calvert<p align=right>Audio
-<p align=left>Mathew C. Burrack<p align=right>Interface audio
-<p align=left>Matthew B. MacLaurin<p align=right>Comportement des avatars
-<p align=left>Paul A. Querna<p align=right>Outils
-<p align=left>Harvey Chapman<p align=right>Systèmes
+				<translation language="French"><![CDATA[<cover src="xUruCreditsJournalCover*1#0.hsm"><font face=Uru size=24 spacing=-10 color=000000><margin right=32 left=32 top=22 bottom=23><p align=left><font size=26><p align=center>Vision<font spacing=-5>
+<font size=24 spacing=-10><p align=left>Rand Miller<p align=right>Directeur
+
+<p align=center>Programmation<font spacing=-5>
+<font size=24 spacing=-10><p align=left>Mark H. DeForest<p align=right>CTO
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:192]: CTO in French is a Directeur de la Technologie, or Directeur Technique

File: compiled/dat/GlobalFrench.loc [hunk 10]
```diff
@@ -185,332 +185,200 @@
...
+<p align=left>Kerryn Miller<p align=right>Responsable du SC
+<p align=left>Taylor Bailey<p align=right>Service Client
+<p align=left>Jesse Bronson<p align=right>Service Client
+<p align=left>Ben Drechsel<p align=right>Service Client
+<p align=left>Chris Lambert<p align=right>Service Client
+<p align=left>Bob Mathias<p align=right>Service Client
+<p align=left>Emily Norton<p align=right>Service Client
+<p align=left>Bradford Chapman<p align=right>Testeur
+<p align=left>John Taitingfong<p align=right>Testeur
+<p align=left>Eliza Wyatt<p align=right>Testeuse
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:242]:
```suggestion
<p align=left>Bradford Chapman<p align=right>Testeur AQ
<p align=left>John Taitingfong<p align=right>Testeur AQ
<p align=left>Eliza Wyatt<p align=right>Testeuse AQ
```

File: compiled/dat/GlobalFrench.loc [hunk 11]
```diff
@@ -185,332 +185,200 @@
...
+<font size=24 spacing=-10><p align=center>Xavier Kemmlein (France)<font spacing=-5>
+<font size=24 spacing=-10><p align=left>Bernard Bollet<p align=right>Atrus
+<p align=left>Nathalie Homs<p align=right>Yeesha
+<p align=left>Antoine Tomé<p align=right>Zandi
 <font spacing=-18>
-<font size=26><p align=center>Musiciens<font spacing=-5>
+
+<pb><font size=26><p align=center>Musiciens<font spacing=-5>
 <font size=24 spacing=-10><p align=left>Eric Engerbretson<p align=right>Guitare
 <p align=left>Friends of Sironka Dance Troupe<p align=right>Choeurs
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:262]:
```suggestion
<p align=left>Friends of Sironka Dance Troupe<p align=right>Chanteurs
```

File: compiled/dat/GlobalFrench.loc [hunk 12]
```diff
@@ -185,332 +185,200 @@
...
-<p align=left>Charlie Adelphia<p align=right>Doudouk
 <p align=left>Tasha Koontz<p align=right>Voix
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:263]:
```suggestion
<p align=left>Tasha Koontz<p align=right>Chanteuse
```

File: compiled/dat/GlobalFrench.loc [hunk 13]
```diff
@@ -185,332 +185,200 @@
...
-<font size=24 spacing=-10><p align=left>Juanjo López<p align=right>Atrus
-<p align=left>Pilar Santigosa<p align=right>Yeesha
-<p align=left>Manuel Bellido<p align=right>Zandi
-<font spacing=-18>
-<font size=24><p align=center>Orange Studio - Bologna (Italie)<font spacing=-5>
-<font size=24 spacing=-10><p align=left>Dario Oppido<p align=right>Atrus
-<p align=left>Grazia Verasani<p align=right>Yeesha
-<p align=left>Lello Lombardi<p align=right>Zandi
-<pb><font size=26><p align=center>Légal (Audio)<font spacing=-5>
+<pb><font size=26><p align=center>Un Merci Spécial à<font spacing=-5>
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:274]:
```suggestion
<pb><font size=26><p align=center>Remerciement Spécial à<font spacing=-5>
```

File: compiled/dat/GlobalFrench.loc [hunk 14]
```diff
@@ -185,332 +185,200 @@
...
+Blake Lewin
+Roger Faso
+Eric Large
+Ron Meiners
+Celia Pearce
+
+<pb><font size=26><p align=center>Légal<font spacing=-5>
 <font spacing=-18>
 <font size=24 spacing=-10><p align=center>Friends of Sironka Dance Troupe
 Certains sons et certains enregistrements vocaux sont fournis par The Friends of Sironka Dance Troupe, sous licence non-exclusive de Cyan, utilisé avec leur autorisation.
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:285]:
```suggestion
Certains sons et enregistrements vocaux sont fournis par The Friends of Sironka Dance Troupe, sous licence nonexclusive de Cyan, utilisé avec leur autorisation.
```

File: compiled/dat/GlobalFrench.loc [hunk 15]
```diff
@@ -185,332 +185,200 @@
...
-Directeur Général Ubi Music
-<pb><font size=26><p align=center>Légal (Outils)<font spacing=-5>
-<font spacing=-18>
-<font size=24 spacing=-10><p align=center>Bink
-Utilise Bink Video. Copyright (C) 1997-2003 par RAD Game Tools, Inc.
 <font spacing=-18>
 <font size=24 spacing=-10><p align=center>FaceGen Modeller 2.1
 Singular Inversions, Inc.
 <font spacing=-18>
 <font size=24 spacing=-10><p align=center>Artbeats Digital Film Library
```
Comment (skyisbluu) [inline compiled/dat/GlobalFrench.loc:291]:
```suggestion
<font size=24 spacing=-10><p align=center>Bibliothèque de films numériques Artbeats
```

**Review by skyisbluu — CHANGES_REQUESTED** [review]

File: compiled/dat/GlobalFrench.loc (hunk 9 above)
Comment (ZarothYe) [reply compiled/dat/GlobalFrench.loc:192]: I checked in-game and "Directeur de la Technologie" will fit properly, so I'll go with that.

File: compiled/dat/GlobalFrench.loc (hunk 15 above)
Comment (ZarothYe) [reply compiled/dat/GlobalFrench.loc:291]: I'm not entirely certain if this should be translated. 🤔 This felt more like a proper name of a project or group, similar to "The Friends of Sironka Dance Group" or "The Web M Project". I don't really know much about Artbeats, though, so I could be wrong. Can anyone else weigh in with their thoughts, please?

Comment (skyisbluu) [reply compiled/dat/GlobalFrench.loc:291]: Checked it out - Artbeats is a company that supplies stock footage for use on many different platforms, including Adobe, gettyimages, shutterstock, so the phrase is descriptive of the digital film library itself, not part of the proper name. I think this translation should be ok :)

Comment (Hoikas) [reply compiled/dat/GlobalFrench.loc:291]: I would leave the name of the company (Artbeats) untranslated but translate the specific resource being cited.

**Comment by Hoikas** [comment]
Is this good to go?

**Comment by skyisbluu** [comment]
Yes, I believe there's nothing more to translate, but I leave the final word to Zaroth.

**Comment by ZarothYe** [comment]
It's complete as far as I'm concerned.

**Pull request merged**